version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.2.3-SNAPSHOT"
```

Comment:
Do we want to retain the `SNAPSHOT` suffix on the `develop` branch? Publish this as rc1 and then immediately add `-SNAPSHOT`?

Reply:
yes, I think we can publish `0.2.3-rcX` and then go back to `0.2.3-SNAPSHOT`.